packages/dev/core/src/scene.ts

```diff
@@ -2650,7 +2651,9 @@ export class Scene implements IAnimatable, IClipPlanesHolder, IAssetContainer {
             newMesh._addToSceneRootNodes();
         }
 
-        this.onNewMeshAddedObservable.notifyObservers(newMesh);
+        DelayCall(() => {

```

Comment:
Is it the next browser event loop, or the next BJS frame we need to wait for? Does waiting for the next browser event loop make assumptions about how the render loop is working? Should this use onAfterRenderObservable.addOnce?

Reply:
We only need to get of the current calling stack (the constructor)